css/style.css

```diff
@@ -565,3 +631,51 @@ a {
   right: 12px;
   top: calc(50% - 24px / 2 - 0.36px);
 }
+
+.rectangle-4-down {
+  background-image: url("/assets/Rectangle-4-down.svg");
+  width: 67.02px;
+  height: 66.58px;
+  left: 82%;
+  top: 12%;
+}
+
+.ellipse-3-down {
+  background-image: url("/assets/Ellipse-3-down.svg");
+  width: 47px;
+  height: 105px;
+  left: 87%;
+  top: 15%;
+}
+
+.ellipse-12-down {
+  background-image: url("/assets/Ellipse-12-down.svg");
+  width: 24.14px;
+  left: 43%;
+  top: 15%;
+}
+
+.ellipse-5-down {
+  background-image: url("/assets/Ellipse-5-down.svg");
+  width: 35px;
+  height: 4%;
+  left: 66%;
+  top: 88%;
+  transform: rotate(52.97deg);
+}
+
+.rectangle-7-down {
+  background-image: url("/assets/Rectangle-7-down.svg");
+  width: 18%;
+  height: 8%;
+  left: 66%;
+  top: 84%;
+}
+
+.indicator {
+  background-image: url("/assets/Shape.svg");
+  width: 36%;
+  height: 1%;
+  left: 33%;
+  top: 99%;
+}
```

Comment:
- [ ] DRY stands for "Don't Repeat Yourself." The main idea with DRY in CSS is to avoid repeating the same bits of code in our style sheet. You are repeating your code multiple times, Let us fix this issue in this way. 
    - Group reusable css properties together.
    - Name these groups logically.
    - Add your selectors to the various css groups.

Reply:
I appreciate you pointing this out, I also understand the idea with DRY in CSS, but the properties here have different units if I try to use a group it will be very rough as I'll be grouping for each property on the code. 😢